### PR TITLE
Adds a review date to every blog post

### PR DESCRIPTION
**Pull request by ABizzinotto**
@etagwerker @lubc I added a `reviewed` column with today's date as the review date to every article. This will give us three months to work on a solution for Blogcop opening too many pull requests at once. This is not a fix. It's a stalling mechanism.

### CODE REVIEW

**Comment by ABizzinotto** [comment]
@lubc thoughts?

File: _posts/2015-08-25-almundo-bill-vulnerability.markdown [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: "Almundo's Bill Vulnerability"
-date: 2015-08-25 03:55:00
+date: 2015-03-25 03:55:00
```
Comment (lubc) [inline _posts/2015-08-25-almundo-bill-vulnerability.markdown]: @ABizzinotto you missed the reviewed date here

File: _posts/2018-06-22-announcing-audit-tool.markdown [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: post
 title:  "Announcing Gemfile.lock Audit Tool"
 date: 2018-06-22 11:10:00
-reviewed: 2020-02-06 7:26:00
+rreviewed: 2020-03-05 10:00:00
```
Comment (lubc) [inline _posts/2018-06-22-announcing-audit-tool.markdown]: @ABizzinotto typo here

File: _posts/2015-08-25-almundo-bill-vulnerability.markdown (hunk 1 above)
Comment (ABizzinotto) [reply _posts/2015-08-25-almundo-bill-vulnerability.markdown]: Added

File: _posts/2018-06-22-announcing-audit-tool.markdown (hunk 2 above)
Comment (ABizzinotto) [reply _posts/2018-06-22-announcing-audit-tool.markdown]: Corrected

**Comment by etagwerker** [comment]
> I guess we can also uninstall blogcop from the repo until we come up with a solution.

@lubc how hard would it be to just create one issue at a time instead of `n` issues?

**Comment by ABizzinotto** [comment]
@lubc did you see @etagwerker 's comment?

**Comment by lubc** [comment]
> @lubc how hard would it be to just create one issue at a time instead of `n` issues?

@etagwerker @ABizzinotto  Sorry for the delay. I estimate it would be around 5 points https://www.pivotaltracker.com/story/show/169424389
Right now the app is pretty basic (https://github.com/ombulabs/blogcop/blob/master/server.rb#L46). It has to be a bit more complex to allow that. I added some items to the story.

**Pull request merged**